File: WebAPI/WebAPI/Models/Flight.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.Text.Json.Serialization;

namespace WebAPI.Models
{
    public class Flight
    {
		public int FlightId { get; set; }
		public string Dtaascend { get; set; }
		public string Dtadescend { get; set; }
		public string Duration { get; set; }
		public double Distance { get; set; }
		public int Ticketprice { get; set; }
		public double Stars { get; set; }
		public List<Stop> Stops { get; set; }
		public List<Seat> Seats { get; set; }

		public int DestinationId { get; set; }
		[JsonIgnore]
		public Destination Destination { get; set; }

		public Flight()
		{

		}
	}
}

File: Angular9/src/entities/flight.ts
import { Stop } from './stop';
import { Seat } from './seat';

export class Flight
{
    flightId: number;
    dtaascend: string;
    dtadescend: string;
    duration: number;
    distance: number;
    ticketprice: number;
    stars: number;
    stops: Array<Stop>;
    seats: Array<Seat>;

    // constructor(dtascend: string, dtdescend: string, duration: number, distance: number, ticketprice: number, stops: Array<Stop>)
    // {
    //     this.distance = distance;
    //     this.dtascend = dtascend;
    //     this.dtdescend = dtdescend;
    //     this.ticketprice = ticketprice;
    //     this.stops = stops;
    //     this.duration = duration;
    // }
    /**
     *
     */
    constructor() {
        this.stops = new Array<Stop>();
        this.seats = new Array<Seat>();
    }
}